lib/engine/action/corporate_sell_shares.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+require_relative 'sell_shares'
+
+module Engine
```

Comment:
Is a separate action actually needed? The entity type says it's a corporation anyway

Reply:
Similar to Without a separate action class, the IssuesShares view is invoked ( see line 40 of view/game/round/operating.rb )

Reply:
couldn't we remove CorporateBuyShares action by just checking a type specific to the round.

it seems strange to create a new action type just for one game when we already have actions for these things.

how clean or difficult is it to just use the existing buy_shares / sell_shares and just have a special flag for 18co

Reply:
buy_shares / sell_shares actions are already used in the Operating Round for the issue shares and redeem shares views. Therefore, we can't base the decision "by checking a type specific to the round" as far as I interpret your suggestion.